app/src/components/tables/ValidatorTable/ValidatorTable.test.tsx

```diff
@@ -0,0 +1,180 @@
+import React from 'react';
+import { screen } from '@testing-library/react';
+import { Table } from 'src/components';
+import {
+  getMockCurrentEraValidators,
+  getMockValidatorsTableFooter,
+  mockValidatorsTableColumns,
+  mockValidatorsTableOptions,
+  getMockNextEraValidators,
+  getMockValidatorsTableHeader,
+} from 'src/mocks/mock-validator-table';
+import { standardizeNumber, truncateHash } from 'src/utils';
+import { standardizePercentage } from 'src/utils/standardize-percentage';
+import userEvent from '@testing-library/user-event';
+import { render } from '../../../test-utils';
+import { EraToggleButton } from './ValidatorTable';
+
+const mockCurrentEraValidators = getMockCurrentEraValidators();
+const mockNextEraValidators = getMockNextEraValidators();
+const mockValidatorsTableHeader = getMockValidatorsTableHeader();
+const mockValidatorsTableFooter = getMockValidatorsTableFooter();
+const totalPages = Math.ceil(
+  mockCurrentEraValidators.status.validatorsCount /
+    mockValidatorsTableOptions.pagination.pageSize,
+);
+
+describe('ValidatorsTable', () => {
+  it('should render Validators Table', () => {
+    render(
+      <Table
+        header={mockValidatorsTableHeader}
+        columns={mockValidatorsTableColumns}
+        data={mockCurrentEraValidators.validators}
+        footer={mockValidatorsTableFooter}
+        tableBodyLoading={false}
+        currentPageSize={mockValidatorsTableOptions.pagination.pageSize}
+        placeholderData={{}}
+        isLastPage={
+          totalPages === mockValidatorsTableOptions.pagination.pageSize
+        }
+      />,
+    );
+
+    const validatorsTableHeader = screen.getByTestId('validators-table-header');
+    const validatorsBaseTable = screen.getByTestId('base-table');
+    const validatorsTableFooter = screen.getByTestId('validators-table-footer');
+
+    expect(validatorsTableHeader).toBeInTheDocument();
+    expect(validatorsBaseTable).toBeInTheDocument();
+    expect(validatorsTableFooter).toBeInTheDocument();
+  });
+
+  it('should render a Rank, a truncated Public Key, a Fee, number of Delegators, Total Stake for Current Era, Self Percentage and Percentage of Network for each validator', () => {
```

Comment:
Let's actually individualize the tests per column value like you had before but just render the table at the top of the test blocks. Does that make sense? Sorry, I should have been more clear from the get go.

Reply:
Made the update using beforeEach(). However, I had to add new describe block for the Next Era -> Total Stake adjustment